CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+We welcome contributions to dws
+
+# Pull Requests
+
+This repository follows the standard GitHub fork+pull model.  We accept GitHub pull requests.
```

Comment:
does it? 😉

Reply:
Yes.  That's how LLNL folks would submit any additions of their own.  The same thing they'd do for nnf-sos or nnf-dm.